docs/tools/slurm.md

```diff
@@ -20,37 +20,103 @@ The following sections will provide detailed guidance on how to use SLURM to req
 [](){#gh200-slurm}
 ### NVIDIA GH200 GPU Nodes
 
-!!! todo
-    document how slurm can be used on the Grace-Hopper nodes.
+The [GH200 nodes on Alps][gh200-node] have four GPUs per node, and SLURM job submissions must be configured appropriately to best make use of the resources. Applications that can saturate the GPUs with a single process per GPU should generally prefer this mode. [Configuring SLURM jobs to use a single GPU per rank][gh200-slurm-single-rank-per-gpu] is also the most straightforward setup. Some applications perform badly with a single rank per GPU, and require use of [NVIDIA's Multi-Process-Service (MPS)](https://docs.nvidia.com/deploy/mps/index.html) to oversubscribe GPUs with multiple ranks per GPU.
```

Comment:
I have a habit of putting one sentence per line with no wrapping in markdown
* the sentences will still be rendered as paragraphs
* it often reduces the chance of nasty merges where two people edit the same paragraph.
    * you are still stuffed if they edit the same sentence.

Reply:
Yep, there are downsides to any approach. I've split up the lines in https://github.com/eth-cscs/cscs-docs/pull/12/commits/7922cd8251af49b4a994ec7d8f0191e9c7af6d9d (for the parts that I changed).